client/src/pages/Balance/ReactStripe.js

```diff
@@ -0,0 +1,304 @@
+  CardElement,
+  Elements,
+  useElements,
+  useStripe
+} from "@stripe/react-stripe-js";
+import { UserContext } from "context/UserContext";
+import { Grid, Button, makeStyles, Typography } from "@material-ui/core";
+import { Link } from "react-router-dom";
+import "./common.css";
+import axios from "axios";
+
+const CARD_OPTIONS = {
+  iconStyle: "solid",
+  style: {
+    base: {
+      iconColor: "#fff",
+      color: "#fff",
+      fontWeight: 500,
+      fontFamily: "Roboto, Open Sans, Segoe UI, sans-serif",
+      fontSize: "16px",
+      fontSmoothing: "antialiased",
+      ":-webkit-autofill": {
+        color: "#fce883"
+      },
+      "::placeholder": {
+        color: "#fff"
+      }
+    },
+    invalid: {
+      iconColor: "#e8977e",
+      color: "#e8977e"
+    }
+  }
+};
+
+const CardField = ({ onChange }) => (
+  <div className="FormRow">
+    <CardElement options={CARD_OPTIONS} onChange={onChange} />
+  </div>
+);
+
+const Field = ({
+  label,
+  id,
+  type,
+  placeholder,
+  required,
+  autoComplete,
+  value,
+  onChange
+}) => (
+  <div className="FormRow">
+    <label htmlFor={id} className="FormRowLabel">
+      {label}
+    </label>
+    <input
+      className="FormRowInput"
+      id={id}
+      type={type}
+      placeholder={placeholder}
+      required={required}
+      autoComplete={autoComplete}
+      value={value}
+      onChange={onChange}
+    />
+  </div>
+);
+
+const SubmitButton = ({ processing, error, children, disabled }) => (
+  <button
+    className={`SubmitButton ${error ? "SubmitButton--error" : ""}`}
+    type="submit"
+    disabled={processing || disabled}
+  >
+    {processing ? "Processing..." : children}
+  </button>
+);
+
+const ErrorMessage = ({ children }) => (
+  <div className="ErrorMessage" role="alert">
+    <svg width="16" height="16" viewBox="0 0 17 17">
+      <path
+        fill="#FFF"
+        d="M8.5,17 C3.80557963,17 0,13.1944204 0,8.5 C0,3.80557963 3.80557963,0 8.5,0 C13.1944204,0 17,3.80557963 17,8.5 C17,13.1944204 13.1944204,17 8.5,17 Z"
+      />
+      <path
+        fill="#6772e5"
+        d="M8.5,7.29791847 L6.12604076,4.92395924 C5.79409512,4.59201359 5.25590488,4.59201359 4.92395924,4.92395924 C4.59201359,5.25590488 4.59201359,5.79409512 4.92395924,6.12604076 L7.29791847,8.5 L4.92395924,10.8739592 C4.59201359,11.2059049 4.59201359,11.7440951 4.92395924,12.0760408 C5.25590488,12.4079864 5.79409512,12.4079864 6.12604076,12.0760408 L8.5,9.70208153 L10.8739592,12.0760408 C11.2059049,12.4079864 11.7440951,12.4079864 12.0760408,12.0760408 C12.4079864,11.7440951 12.4079864,11.2059049 12.0760408,10.8739592 L9.70208153,8.5 L12.0760408,6.12604076 C12.4079864,5.79409512 12.4079864,5.25590488 12.0760408,4.92395924 C11.7440951,4.59201359 11.2059049,4.59201359 10.8739592,4.92395924 L8.5,7.29791847 L8.5,7.29791847 Z"
+      />
+    </svg>
+    {children}
+  </div>
+);
+
+const ResetButton = ({ onClick }) => (
+  <button type="button" className="ResetButton" onClick={onClick}>
```

Comment:
Where are the SubmitButton or ResetButton components defined here used? I can't find it.